joomla/joomla-cms
# [4.0] Dependency - Bump joomla/filter to 2.0.0-beta5

Pull Request for Issue # .

### Summary of Changes

Update the joomla/filter dependency to 2.0.0-beta5.

### Testing Instructions

Check that the CMS continues to function as normal.
In particular, check that:
- Filtering works as usual for fields.
- Saving Global Configuration works.

Check that CI tests succeed.

### Actual result BEFORE applying this Pull Request

CMS works. CI tests succeed.

### Expected result AFTER applying this Pull Request

CMS works. CI tests succeed.

### Documentation Changes Required

None.
